packages/venia-concept/src/RootComponents/Category/categoryContent.js

```diff
@@ -21,13 +45,26 @@ class CategoryContent extends Component {
                         }}
                     />
                     <div className={classes.categoryTitle}>{categoryTitle}</div>
+                    <div className={classes.headerButtons}>
+                        <button
```

Comment:
We should conditionally render this button based on the `filters` value from data. We already return null within the modal if there are no filters, so we shouldn't even display the button in that case :)

Reply:
Fixed in latest commit.